test/case/infix_services/services_basic.py

```diff
@@ -72,24 +85,33 @@ def verify(enabled, sec):
                 "enabled": False
             }
         })
+        target.put_config_dict("ieee802-dot1ab-lldp", {
+            "lldp": {
+                "enabled": False
+            }
+        })
         cfg = target.get_config_dict("/infix-services:ssdp")
         assert not cfg["ssdp"]["enabled"]
```

Comment:
For symmetry, should we add the same checks for mDNS and LLDP?

Reply:
Maybe, or drop and in the future replace with an operational check?

Reply:
Even better.